java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -371,4 +371,22 @@ CompletableFuture<Boolean> pexpireAt(
      * }</pre>
      */
     CompletableFuture<String> type(String key);
+
+    /**
+     * Returns the internal encoding for the Redis object stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/object-encoding/">redis.io</a> for details.
+     * @param key The <code>key</code> of the object to get the internal encoding of.
+     * @return If <code>key</code> exists, returns the internal encoding of the object stored at
+     *     <code>key</code> as a <code>String</code>. Otherwise, return <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * String encoding = client.objectEncoding("key").get();
+     * assert encoding.equals("encoding");
+     *
+     * String encoding = client.objectEncoding("non_existing_key").get();
```

Comment:
I would rename this variable, so the code actually compiles as is.  

Or better yet, just remove the negative example.  The examples are to demonstrate how the commands work, and this only shows a user what _not_ to do.

Reply:
I copied this from the "get" command in our StringBaseCommands 👀

Reply:
The implication was more that we probably should make the `get` command example compile.

Reply:
that would be good idea